### PR TITLE
@kanaabe => Publishing video margin

### PR DESCRIPTION
**Pull request by damassi**
Addresses 
- https://github.com/artsy/publishing/issues/200
- https://github.com/artsy/publishing/issues/189

https://www.artsy.net/article/artsy-editorial-happened-laurie-simmonss-dolls-life

screen shot 2017-11-04 at 2 00 13 am

### CODE REVIEW

**Comment by damassi** [comment]
Going to self-merge to get this into staging testing

File: src/Components/Publishing/Header/BasicHeader.tsx [hunk 1]
```diff
@@ -148,27 +149,24 @@ const Container = styled.div`
...
       height: ${800 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.sm`
       height: ${620 * VIDEO_RATIO}px;
     `}
-
     ${breakpoint.xs`
-      height: ${520 * VIDEO_RATIO}px;
+      height: ${340 * VIDEO_RATIO}px;
```
Comment (kanaabe) [inline src/Components/Publishing/Header/BasicHeader.tsx]: Now that the 🔥  is simmered down, can you remind me again why we made these breakpoints have fixed values? Maybe it's worth a quick SH but unless these values were specific to the design, I'd ideally like to replace these breakpoints with -- `height: calc((100vw - 100) * ${VIDEO_RATIO})` or some dynamic variation of that is possible.

File: src/Components/Publishing/Sections/Sections.tsx [hunk 2]
```diff
@@ -200,4 +200,8 @@ const StyledSections = Div`
   ${props => pMedia.md`
     max-width: ${props.layout === "standard" ? "780px" : "auto"};
   `}
+  ${props => pMedia.xs`
+    max-width: ${props.layout === "standard" ? "780px" : "auto"};
+    ${props.layout === 'feature' ? "margin: 30px auto 0 auto" : ""}
```
Comment (kanaabe) [inline src/Components/Publishing/Sections/Sections.tsx]: Minor - missing a `;` at the end of this.

File: src/Components/Publishing/Header/BasicHeader.tsx (hunk 1 above)
Comment (damassi) [reply src/Components/Publishing/Header/BasicHeader.tsx]: These values were computed by Owen and represent the width * a fraction, which returns the proper aspect-ratio-height per the designs. I can try the `vh` approach but note that these values now fit the spec nicely!

**Pull request merged**